src/docs/samples/primer/kotlin/settings.gradle.kts

```diff
@@ -1 +1,26 @@
-pluginManagement.includeBuild("gradle/plugins")
+pluginManagement {
+    includeBuild("gradle/plugins")
+}
```

Comment:
I was surprised that the `pluginManagement.includeBuild("gradle/plugins")` worked at all (and does not fail the build). It seems to silently delegate to the "normal" `inculdeBuild(...)` (??). In any case, using it leads to the plugins not being found in _settings.gradle.kts_.

`pluginManagement {}` is treated as a separate block (at least in Kotlin), similar as `plugins {}`. I think all the config needs to be in the closure there.